apps/L1G/restaurant/restaurant-frontend/src/components/auth/update-user-profile/EditPasswordDialog.tsx

```
@@ -0,0 +1,105 @@
+      confirmPassword: '',
+    },
+  });
+
+  const onSubmit = async (data: PasswordFormValues) => {
+    setUpdating(true);
+    try {
+      await onUpdate(data.currentPassword, data.newPassword);
+      setOpen(false);
+      form.reset();
+    } catch (error) {
+    } finally {
```

ESLint

[error] Empty block statement.

(no-empty)